src/ps2.js

```diff
@@ -331,6 +345,20 @@ PS2.prototype.send_mouse_packet = function(dx, dy)
     this.mouse_buffer.push(info_byte);
     this.mouse_buffer.push(delta_x);
     this.mouse_buffer.push(delta_y);
+    if (this.mouse_id == 0x04) {
+        this.mouse_buffer.push(
+			0 << 7 |
+			0 << 6 |
+			0 << 5 | // TODO: 5th button
+			0 << 4 | // TODO: 4th button
+			this.wheel_movement
```

Comment:
Should this be masked to the lower four bits? Otherwise the upper bits are set if wheel_movement is negative.